io.catenax.traction_battery_code/1.0.0/metadata.json

```diff
@@ -1 +1 @@
-{ "status" : "release"} 
+{ "status" : "standardize"} 
```

Comment:
BAMM to SAMM conversion required for this model.

Reply:
@agg3fe can we do the migration from BAMM to SAMM wihtin this PR or need we to open a new issue incl. a new PR? (We prefer the first option, since a new issue and PR for the migrate produce unnecessary work for us)

Reply:
@johannsvarela is this model already used by someone? Usually We create a new PR for BAMM to SAMM change and this is a major version change and not backward compatible. Only if it is not used by anyone, then we might do the change in same model.

Reply:
@agg3fe Since this model isn't standardized yet, it should be used by anyone. @jonbckr @jacewski-bosch Am i right with my assumption? :-)